public/app/tour/steps.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { StepType } from '@reactour/tour';
+import SidebarStep from './SidebarStep';
+
+const steps: StepType[] = [
+  {
+    selector: '.dropdown > [aria-label="PMM dashboards"]',
+    content: (
+      <SidebarStep title="PMM Dashboards">
+        <p>
+          PMM ships with several Grafana dashboards for efficient database monitoring, from CPU utilization to MySQL
+          Replication Summary.
+        </p>
+        <p>
+          Check our{' '}
+          <a
+            href="https://github.com/percona/grafana-dashboards"
+            target="_blank"
+            rel="noreferrer noopener"
+            style={{ textDecoration: 'underline' }}
+          >
+            dashboards repo
+          </a>{' '}
+          and see how you can contribute with your own.
+        </p>
+      </SidebarStep>
+    ),
+  },
+  {
+    selector: '.dropdown > [aria-label="Alerting"]',
+    content: (
+      <SidebarStep title="PMM Alerting">
+        <p>PMM comes with a simpler-to-use alerting system that works side-by-side with Grafana&apos;s.</p>
+        <p>
+          You define what system metrics are critical to your environment, and what thresholds are acceptable for each
```

Comment:
I understand why you didn't put all this text to separate constants, but if we decide to support multiple languages in pmm in the future, it would be much preferable to have all texts in a separate file as in other places.

Reply:
Yeah, good point 👍